### ISSUE TITLE
Add map obstacles

### ISSUE THREAD

**Issue by josefnpat** [josefnpat/roguecraft-squadron#680]
Feedback:

> I would have liked to see more obstacles on the map, instead of it being wide open

**Comment by softsound** [comment]
Yeah, we have clouds, black holes, but I would like garbage piles and what else do you find in space?
